tensorflow_addons/custom_ops/image/cc/kernels/connected_components.h

```diff
@@ -0,0 +1,305 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+// See docs for ImageConnectedComponents in ../ops/image_ops.cc, and description
+// of the algorithm in connected_components.h.
+
+#ifndef TENSORFLOW_CONTRIB_IMAGE_KERNELS_SEGMENTATION_OPS_H_
+#define TENSORFLOW_CONTRIB_IMAGE_KERNELS_SEGMENTATION_OPS_H_
```

Comment:
Better to modify this. I'd vote for `TENSORFLOW_ADDONS_CONNECTED_COMPONENTS_H_`. @seanpmorgan Hi Sean, is there any naming convention on this? Thanks!